js/example-frontend-js/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
Should this be changed to 2021?

Reply:
Practice, used in Kotlin - update year in all files in separate commit